code/game/objects/items/weapons/defib.dm

```diff
@@ -398,7 +398,7 @@
 						for(var/obj/item/organ/external/O in H.organs)
 							total_brute	+= O.brute_dam
 							total_burn	+= O.burn_dam
-						if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !ghost && tplus < tlimit && !(NOCLONE in H.mutations) && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)))
+						if(total_burn <= 180 && total_brute <= 180 && !H.suiciding && !H.hellbound && !ghost && tplus < tlimit && !(NOCLONE in H.mutations) && (H.get_int_organ(/obj/item/organ/internal/heart) || H.get_int_organ(/obj/item/organ/internal/brain/slime)))
```

Comment:
I'll look into that, @Crazylemon64, I was planning on just shunting them into a mob in hell once they die, but a proc might work just as well.

Hell, why not both?